docs/nodes/credentials/IMAP/README.md

```diff
@@ -44,7 +44,26 @@ Create an email account on a service with IMAP support.
 
 **Note:** Before executing the node, set the ***Ignore SSL Issues*** option to true as the Gmail certificate is self-signed.
 
+## Using Yahoo Mail
```

Comment:
@harshil1712 Can you also please add a video here like with the other credentials?